src/main/java/com/solace/samples/jcsmp/features/SecureSession.java

```diff
@@ -151,6 +156,19 @@ void createSession(String[] args) throws InvalidPropertiesException {
         if (conf.getSslConnetionDowngrade() != null){
         	properties.setProperty(JCSMPProperties.SSL_CONNECTION_DOWNGRADE_TO, conf.getSslConnetionDowngrade());
         }
+        // option for in-memory trust stores and key stores:  https://docs.solace.com/API-Developer-Online-Ref-Documentation/java/com/solacesystems/jcsmp/JCSMPProperties.html
+        if (conf.getTrustStore() == null && conf.getTrustStorePwd() == null) {
+            try {
+                KeyStore ks = KeyStore.getInstance(KeyStore.getDefaultType());
+                char[] password = "some password".toCharArray();

```

Comment:
add comment to retrieve password in a manner consistent with how they import other secrets. don't hard code.